Cargo.toml

```diff
@@ -73,7 +73,7 @@ assert_fs = "1.1.2"
 # polkadot-sdk and friends
 codec = { version = "3.6.12", default-features = false, package = "parity-scale-codec" }
 scale-info = { version = "2.11.6", default-features = false }
-polkadot-sdk = { git = "https://github.com/paritytech/polkadot-sdk", rev = "cb0d8544dc8828c7b5e7f6a5fc20ce8c6ef9bbb4" }
+polkadot-sdk = { git = "https://github.com/paritytech/polkadot-sdk", branch = "at/polkavm" }
```

Comment:
why branch, and not rev?

Reply:
It is a companion to this PR. Wil lbe changed to a master ref once the associated PR is merged.